appsTransfer/src/main/java/org/kinecosystem/appstransfer/presenter/SenderServiceBinder.kt

```diff
@@ -73,11 +81,12 @@ class SenderServiceBinder(private val context: Context?) {
         if (isBounded) {
```

Comment:
isBound should be declared volatile and checked inside the thread. If you check it before executing the thread, by the time the thread is started it could possibly change (edge case but possible)

Reply:
There is code duplication here and in AppInfoActivity - Please make sure that both activities reuse the same code instead of duplicating